### ISSUE TITLE
Import Tachyons or individual modules?

### ISSUE THREAD

**Issue by anater** [juanparadox/react-tachyons#26]
Should we import individual modules into each component to leverage Webpack better? We may be able to save on file size when pieces of Tachyons that aren't used don't get downloaded.

CONS:
- limits prototyping in the browser with all Tachyons classes at disposal
- may result in duplicate requests on the same import (not sure how webpack handles this)
- super cumbersome to refactor for this

**Comment by juanparadox** [comment]
I agree with you in that we should leverage Webpack better, because, after all, the Tachyons team explicitly states that importing just what you need is what makes it so light. Therefore, I believe we should stay as light as we can. Each component would only import it's own CSS.

I believe webpack only ever loads what is needed then and there, but still, caching will help prevent the "load" on the front end. I may be wrong since I'm a noob though.

At this point it does seem cumbersome to refactor. We can definitely revisit this, though, when we have everything set up and ready to start prototyping with.

**Comment by anater** [comment]
Sounds good. Step 1, get the components done. Step 2, figure this out lol

**Comment by anater** [comment]
Accidentally closed this one. Should I leave it open for when we get to this change? Or should we open a new issue to make the change? Does it even matter? Lol
